index.html

```diff
@@ -12,6 +13,43 @@
         type="text/javascript"></script>
     <script src="http://code.jquery.com/mobile/git/jquery.mobile-git.js"
         type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js" type="text/javascript"></script>
```

Comment:
You're right, ideally we shouldn't do it either, especially considering I've been bitten by changes made to it a couple of times